src/pwa-buildpack/reference/magento-resolver/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: MagentoResolver
+---
+
+An adapter that configures Webpack to resolve assets using Magento PWA conventions.
+
+This module generates a configuration object used in the [`resolve`] property of a Webpack config.
+The configuration object tells Webpack how to traverse the filesystem structure for assets required in source files.
+
+Currently, MagentoResolver does very little, but it's likely that the Magento development environment will require custom resolution rules in the future; this utility sets the precedent of the API for delivering those rules.
+
+## API
+
+### `configure(options)`
+
+**Parameters:**
+
+* `options: ResolverOptions`
+    * `options.path: object`(**required**) - Local absolute paths to theme folders
+    * `options.root` - Absolute path to the root directory of the theme
+
+**Return**
+
+[Promise] configuration type for Webpack
+
+{: .bs-callout .bs-callout-info}
+**Note:**
+`MagentoResolver.configure()` is asynchronous.
+
+## Example
+
+In `webpack.config.js`:
+
+``` js
+const buildpack = require('@magento/pwa-buildpack');
+const MagentoResolver = buildpack.Webpack.MagentoResolver;
```

Comment:
I originally wrote this with destructuring, which made it harder to read for beginners. Thanks for this subtle but very nice change.